app/server/utils/ussd/kenya_ussd_state_machine.py

```diff
@@ -139,33 +145,50 @@ def complete_initial_pin_change(self, user_input):
     def complete_pin_change(self, user_input):
         change_current_pin(user=self.user, new_pin=user_input)
 
-    def recipient_exists(self, phone_number):
-        pass
+    # recipient exists, is not initiator, matches active and agent requirements
+    def is_expected_recipient(self, user, should_be_active, should_be_agent):
```

Comment:
very minor but the original `is_valid_recipient` feels like a better naming to me - took me a couple of seconds to grok that we weren't _expecting_ a particular recipient per se.

Reply:
i changed it because it's not necessarily a valid recipient. in the case of upsell, we're looking to match specifically on an `is_disabled` user. but i'll change the name back, expected is confusing you're right